packages/test/test-end-to-end-tests/src/test/idCompressor.spec.ts

```diff
@@ -139,6 +139,40 @@ describeNoCompat("Runtime IdCompressor", (getTestObjectProvider) => {
 		assert(getIdCompressor(map) === undefined);
 	});
 
+	it("can't enable compressor on an existing container", async () => {
```

Comment:
I missed this in my last comment - You should another test that validates it cannot be disabled once enabled.

Reply:
Nice! Maybe rename this test title for consistency with the other test.